winui/Ribbon/RibbonScreenTip.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: ScreenTip | WinUI | Ribbon | Syncfusion
+description: Learn more about the Quick Access ToolBar  in the Syncfusion WinUI Ribbon (SfRibbon) control.
+platform: winui
+control: Ribbon
+documentation: ug
+---
+
+# ScreenTip in WinUI Ribbon Control
+
+`ScreenTip` is an enhanced ToolTip and it is used to show a popup window when the mouse cursor is hovered over `RibbonItem`. The popup is used to provide details of the `RibbonItem’s` function. The ScreenTip consist of two parts `Title` and `Content`. User can customize the ScreenTip title and content by `TitleTemplate` and `ContentTemplate` properties.
+
+## Adding ScreenTip for ribbon items
+
+ScreenTip can be added as follows.
+

```

Comment:
ScreenTip can be set to each Ribbon item by using its `ScreenTip` property. The below code shows how to define the ScreenTip for ribbon items in Ribbon control.

Reply:
resolved